samples/wifi/radio_test/sample_description.rst

```diff
@@ -130,7 +130,7 @@ Testing
               wifi_radio_test tx 0
               wifi_radio_test tx_pkt_tput_mode 0
               wifi_radio_test tx_pkt_len 4095
-              wifi_radio_test chnl_primary 14
+              wifi_radio_test chnl_primary 1
```

Comment:
API name to set Channel number is changed from chnl_primary to init. Please correct.